src/main/java/newRacingCar/domain/Car.java

```diff
@@ -0,0 +1,43 @@
+package newRacingCar.domain;
+
+import java.util.Objects;
+
+public class Car {
+    private Name name;
+    private Position position = new Position(0);
+
+    Car(Name name) {
+        this.name = name;
+    }
```

Comment:
```suggestion
    Car(Name name) {
        this(name, new Position(0));
    }
```
` Car(Name name, Position position)` 을 테스트에서만 쓰고 있는데요, `Car(Name name)` 에서 이를 재사용하는 식으로 쓰면 더 좋을 것 같아요 :) 
주 생성자를 정하고 이를 재사용하는 식으로 쓴다면 어떤 장점이 있을까요?